### ISSUE TITLE
error TS2430: Interface 'ConfirmProps' incorrectly extends interface 'ModalProps'.

### ISSUE THREAD

**Issue by almost** [Semantic-Org/Semantic-UI-React#1618]
### Steps

Import stuff from a file in project:

```
import { Grid } from 'semantic-ui-react';
```

Compile with Typescript

### Expected Result

It compiles

### Actual Result

It fails to compile. With error:

```
ERROR in /**REDACTED**/node_modules/semantic-ui-react/dist/commonjs/addons/Confirm/Confirm.d.ts
(4,18): error TS2430: Interface 'ConfirmProps' incorrectly extends interface 'ModalProps'.
  Property 'content' is optional in type 'ConfirmProps' but required in type 'ModalProps'.
```

### Version
0.68.1 has bug, 0.68.0 does not

### Testcase

See above

**Comment by levithomason** [comment]
There's a PR open for this, #1611.  Duplicate https://github.com/Semantic-Org/Semantic-UI-React/issues/1612. Thanks for the report.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.2`.
